ale_linters/hack/hhast.vim

```diff
@@ -26,10 +26,15 @@ function! ale_linters#hack#hhast#GetExecutable(buffer) abort
     return !empty(l:absolute) ? l:absolute : ''
 endfunction
 
+function! ale_linters#hack#hhast#GetInitializationOptions(buffer) abort
+    return {'lintMode': 'open-files'}
```

Comment:
I'll add this in now. Later I plan to set up a default `initialization_options_callback` which reads `ale_FILETYPE_LINTER_options` by default, and then I'll move this into an option and document it.